### PR TITLE
Star Education Society 

### PR DESCRIPTION
**Pull request by Salarhabibi-amir**
I have customized the Capstone template for the Star Education Society which is an Academic organisation. I tried to make the design similar to the Capstone template. I have create only the index and about page. I used javascript to create the Feature Teacher section dynamic.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,115 @@
...
+
+Pleas review my project and make me happy with your nice feedbacks.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank from Microvers Because of preparing a great envirnment for everyone in all over the globe.
```
Comment (IngDavidTM) [inline README.md]: - [ ] Kindly, as a project requirement, the README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :+1: :100:.

File: style.css
Comment (IngDavidTM) [inline style.css:1]: - [ ] While you did a fantastic job :+1:, as a project requirement the pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design.
**Hint:** Basic color swatch: #272A31, #EC5242, #D3D3D3

File: about.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        </div>
+        </div>
+    </nav>
+
+    <!-- Headline section -->
+    <div class="container-fluid bg-light">
+        <div class="container">
+            <div class="text-center py-2">
+                <h3 class="text-danger py-2 d-md-none d-sm-block">"Hello Sharing World!"</h3>
+                <h1 class=" fw-red fw-bolder">STAR EDUCATIONAL <br>SOCEITY 2023</h1>
```
Comment (IngDavidTM) [inline about.html:92]: - [ ] Great work on recognizing the titles within your project :+1:. However, it is important to note that for the sake of semantic structure on your webpage, it is recommended to have only a single `h1` tag per page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+            <br><br>
+            <div class="row text-center">
+                <div class="col">
+                    <h4>Main Program</h4>
+                    <hr id="hr">
+                </div>
+            </div>
+            <div class="program">
+                <div class="program-item text-center bg-cutome p-2">
+                    <div class="icon"><img src="assets/images/discourse.png" alt=""></div>
```
Comment (IngDavidTM) [inline index.html]: - [ ] You've done an excellent job meeting the project requirement of adding images :+1:. However, it is crucial to ensure that your webpage is accessible to all users. To achieve accessibility, please review the images on both the index.html and about.html pages to ensure they have proper alt attributes. These alt attributes provide alternative text descriptions for the images, making them accessible to individuals who may have visual impairments or are using assistive technologies.

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Salarhabibi-amir  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Salarhabibi-amir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                </div>
+
+                <button class="btn btn-light d-block d-md-none border border-1 border-dark my-2 py-4 fw-bolder" id="moreBtn">See More &nbsp;&nbsp;<i class="fa fa-angle-down fw-red"></i></button>
+                <button class="btn btn-light d-block d-md-none border border-1 border-dark my-2 py-4 d-none fw-bolder" id="lessBtn">See Less &nbsp;&nbsp;<i class="fa fa-angle-up fw-red"></i></button>
+
+            </div>
+        </div> 
+       `;
+featureSection.append(featureContainer);
+const seeMoreBtn = document.getElementById('moreBtn');
```
Comment (MohamedRoshdy2021) [inline script.js:136]: there is a small issue that needs to be Fixed over here

**Pull request merged**